bundles/org.openhab.core.model.script/src/org/openhab/core/model/script/jvmmodel/ScriptJvmModelInferrer.xtend

```diff
@@ -76,19 +94,29 @@ class ScriptJvmModelInferrer extends AbstractModelInferrer {
 			}
 		]
 
-		 itemRegistry?.items?.forEach[ item |
-		 	val name = item.name
-				if (fieldNames.add(name)) {
-		 		members += 	script.toField(item.name, script.newTypeRef(item.class)) [
-			 		static = true
-			 	]
-		 	} else {
-		 		logger.warn("Duplicate field: '{}'. Ignoring '{}'.", item.name, item.class.name)
-		 	}
-		 ]
+        itemRegistry?.items?.forEach[ item |
```

Comment:
Please use the same indentation everywhere in this file instead of mixing tabs and spaces.

Reply:
I now applied the formatter on the full file, which touched most lines.